Airports/EGKB/Stars.txt

```diff
@@ -43,7 +43,4 @@ STAR:EGKB:03:#GODLU1H:SAM BIDVA EVEXU SOXUX OKVAP GODLU ELMIV LCE11 LCE12 LCE13
 STAR:EGKB:21:#JACKO1B:SUMUM LOGAN JACKO NONVA BABKU LCE21 LCE22 LCE23 RAVSA GAPGI ATPEV LCE07 OSVEV
 STAR:EGKB:03:#JACKO1B:SUMUM LOGAN JACKO NONVA BABKU LCE21 LCE22 LCE23 RAVSA GAPGI ATPEV LCE07 OSVEV
 STAR:EGKB:21:#JACKO1D:XAMAN LOGAN JACKO NONVA BABKU LCE21 LCE22 LCE23 RAVSA GAPGI ATPEV LCE07 OSVEV
-STAR:EGKB:03:#JACKO1D:XAMAN LOGAN JACKO NONVA BABKU LCE21 LCE22 LCE23 RAVSA GAPGI ATPEV LCE07 OSVEV
-;Withdrawn 2017/12
-STAR:EGKB:21:#JACKO1L:WAL LISTO PEDIG ROGBI TIXEX ODVOD ROPMU NUDNA INLIM JACKO NONVA BABKU LCE21 LCE22 LCE23 RAVSA GAPGI ATPEV LCE07 OSVEV
-STAR:EGKB:03:#JACKO1L:WAL LISTO PEDIG ROGBI TIXEX ODVOD ROPMU NUDNA INLIM JACKO NONVA BABKU LCE21 LCE22 LCE23 RAVSA GAPGI ATPEV LCE07 OSVEV
```

Comment:
JACKO1L at EGKB should remain (it's 2017/12)

Reply:
No worries, just couldn't find anything to do with 2016/02 in there :)